lib/barbeque/message_handler/job_execution.rb

```diff
@@ -20,9 +20,15 @@ def run
           raise DuplicatedExecution.new(e.message)
         end
 
-        stdout, stderr, status = run_command
+        begin
+          stdout, stderr, status = run_command
+        rescue Exception => e
```

Comment:
I guess you chose not `StandardError` but `Exception` to cover `SignalException`, for the case a job gets stuck and signal is sent to kill it. (I just wanted to know the background.)